src/utils/mod.rs

```diff
@@ -1,5 +1,9 @@
+/// Module that contains various methods to comput price
 pub mod compute_price_impact;
+
+/// Module that contains various methods to sort
 pub mod sorted_insert;
+/// Module that contains various methods to compute square root
```

Comment:
```suggestion
pub mod compute_price_impact;
pub mod sorted_insert;
```
Aren't even accurate.